DraftSimPackage.js

```diff
@@ -348,14 +347,29 @@ class DraftSimPackage {
       let predURL = this.masterHash["index_to_url"][pred];
       for (let i = 0; i < displayedPack.length; i++) {
         if (displayedPack[i].src === predURL) {
-          displayedPack[i].style.border = "thick solid #33cc33";
+          //using ID instead of classname here because it's easier + faster to remove between card draws:
+          displayedPack[i].id = "greenSelect";
         }
```

Comment:
this is a pr comment idk if you've ever made one. You add them under the lines you're referencing to ask questions or point out issues